styleguide/_themes/derek/scss/components/mainnav.scss

```diff
@@ -0,0 +1,354 @@
+//main nav
+.main-nav {
+  background: $dark-gray;
+  border: 0;
+  border-radius: 0;
+  margin: 0;
+
+  .navbar-menuContainer {
+    border: 0;
+    max-height: none;
+    padding: 0;
+  }
+}
+
+.navbar-nav {
```

Comment:
@bighappyface Unfortunately navbar behavior isn't available as a mixin 
(see: https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_navbar.scss)

We could theoretically create a mixin for our own personal use to emulate some of that, but that may be better suited for another PR.  @SyntaxStacks is already doing a little bit of that in his basement PR for containers, so we could maybe tie into that down the road. 

Thoughts?